cohortextractor2/concepts/tables.py

```diff
@@ -21,30 +22,40 @@ def __init__(self):
         super().__init__(Table("practice_registrations"))
 
 
-class Patients(TableContract):
+class PatientDemographics(TableContract):
     """
-    The Patients table holds personal details about the patient.
+    Provides demographic information about patients
     """
 
     patient_id = Column(
         type=types.PseudoPatientId(),
-        help=(
+        description=(
             "Patient's pseudonymous identifier, for linkage. You should not normally "
             "output or operate on this column"
         ),
+        help="",
+        constraints=[],
     )
     date_of_birth = Column(
         type=types.Date(),
-        help="The patient's date of birth",
+        description="Patient's year and month of birth.",
+        help="The day will always be the first of the month. Must be present.",
+        constraints=[DateConstraint(match_format=["%Y-%m-01"])],
     )
     sex = Column(
-        type=types.Choice("F", "M"),
-        help="The patient's sex",
+        type=types.Choice("female", "male", "intersex", "unknown"),
+        description="Patient's sex.",
+        help=(
+            "One of male, female, intersex or unknown (the last covers all other options,"
+            "including but not limited to 'rather not say' and empty/missing values). "
+            "Must be present."
+        ),
+        constraints=[ChoiceConstraint()],
```

Comment:
Can this be implied by the type, rather than having to duplicate it?

Reply:
yes, will do that